lib/restaurant.rb

```diff
@@ -0,0 +1,49 @@
+class Restaurant
```

Comment:
Division of class responsibilities is really clear and I like that its all run logically from a 'restaurant' interface